inspector/scripts/utils.ts

```diff
@@ -13,3 +12,76 @@ export function getKeyHash() {
     throw new Error(`Key Hash is not defined for network: ${networkName}`);
   }
 }
+
+export async function readAll(inspectorConsumer: Contract) {
+  const { sRandomWord, vrfRequestId } = await readVrf(inspectorConsumer);
+  const { sResponse, rrRequestId } = await readRr(inspectorConsumer);
+  return { sResponse, sRandomWord, rrRequestId, vrfRequestId };
+}
+
+export async function readVrf(inspectorConsumer: Contract) {
+  const sRandomWord = await inspectorConsumer.sRandomWord();
+  const vrfRequestId = await inspectorConsumer.vrfRequestId();
+  return { sRandomWord, vrfRequestId };
+}
+export async function readRr(inspectorConsumer: Contract) {
+  const sResponse = await inspectorConsumer.sResponse();
+  const rrRequestId = await inspectorConsumer.rrRequestId();
+  return { sResponse, rrRequestId };
+}
+
+export async function requestAll(
+  ACC_ID: string,
+  explorerBaseUrl: string,
+  inspectorConsumer: Contract,
+  networkName: string
+) {
+  await requestVrf(ACC_ID, explorerBaseUrl, inspectorConsumer, networkName);
+  await requestRr(ACC_ID, explorerBaseUrl, inspectorConsumer);
+}
+
+export async function requestVrf(
+  ACC_ID: string,
+  explorerBaseUrl: string,
+  inspectorConsumer: Contract,
+  networkName: string
+) {
+  const keyHash = getKeyHash(networkName);
+  const callbackGasLimit = 500_000;
+  const numWords = 1;
+
+  const vrfTx = await (
+    await inspectorConsumer.requestVRF(
+      keyHash,
+      ACC_ID,
+      callbackGasLimit,
+      numWords
+    )
+  ).wait();
+
+  if (vrfTx.status == 1) {
+    console.log("VRF request: SUCCESS");
+    console.log(`${explorerBaseUrl}/${vrfTx.hash}`);
```

Comment:
This is my fault, but I think it would be good to print the link to blockchain explorer even for the error case to help with potential debugging.

```typescript
if (vrfTx.status == 1) {
    console.log("VRF request: SUCCESS");
} else {
    console.error("VRF request: FAILURE");
}
console.log(`${explorerBaseUrl}/${vrfTx.hash}`);
```

It applies to RR as well.